onnxruntime/hosting/include/beast_http.h

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include <functional>
+#include <iostream>
+#include <memory>
+#include <string>
+#include <thread>
+#include <vector>
+
+#include "util.h"
+#include "http_context.h"
+#include "routes.h"
+#include "http_session.h"
+#include "listener.h"
+
+namespace http = beast::http;      // from <boost/beast/http.hpp>
+namespace net = boost::asio;       // from <boost/asio.hpp>
+using tcp = boost::asio::ip::tcp;  // from <boost/asio/ip/tcp.hpp>
+using handler_fn = std::function<void(std::string, std::string, std::string, Http_Context&)>;
+
+// Accepts incoming connections and launches the sessions
+class App {
+ public:
+  App() {
+    address_ = boost::asio::ip::make_address_v4("0.0.0.0");
+    port_ = 8080;
```

Comment:
default value should come from a central place. But TODO for this quick PR to enable boost::beast is acceptable to me.

Reply:
Added a todo